md/dates-management-tutorial.md

```diff
@@ -0,0 +1,221 @@
+# Manage dates in BDM and User Interfaces
+
+::: info
+This howto applies to Bonita BPM versions prior to 7.5.0  
+:::
+
+Up to versions 7.4.x, only date selection is supported natively in the UI Designer using the **Date picker** widget. So this page will only focus on date management (not date and time).
+
+::: warning
+If you only use the API to retrieve dates, and you do not need to query the business data database directly, you can safely use the type **DATE** for your business objects.
+If you query the database directly, use the type **STRING** instead of **DATE**. Otherwise the time zone of the server will be used when converting the java Date in an SQL **TIMESTAMP** and, unless your server is configured in UTC, an offset will be applied. Therefore, the value stored in the database will not reflect the date entered by the user.
+:::
+
+The following sections show how to create, update, and display an *Employee* object with a birth date stored either as a DATE or a STRING in the BDM.  
+
+## Business object attributes with type DATE
+
+#### Define the business data
+
+1. In the studio, open the Business Data Model editor (in menu **Development** > **Business Data Model** > **Manage...**)
+1. Create a **business object** *Employee* with an attribute *birthdate* of type **DATE**. (It uses `java.util.Date` as Java type)  
+1. Create a new process
+1. In the process **Data** tab, create a new **business variable** *employee* of type *Employee*
+
+#### Define the contract at pool level, as well as the script to initialize the business variable
+
+1. In the process **Execution** tab > **Contract** section, define the contract inputs using the **Add from data...** button
+1. Select the business variable *employee*
+1. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **DATE** (keep the option to generate the variable initialization script automatically)
+
+![Contract](images/dates-management-tutorial/tuto-dates-contract-date.png) <!--{.img-responsive .img-thumbnail}-->
```

Comment:
the contract shows more attributes than expected from the business data chapter.
May be worth adjusting